openslides_backend/models/motion_category.py

```diff
@@ -0,0 +1,16 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionCategory(Model):
+    """
+    Model for motion category.
+    """
```

Comment:
Please add docstring with all reverse fields (or as TODO sentence).

Reply:
Add the two reverse fields to the docstring.

Reply:
Please follow the syntax from the `models.txt`, e.g.

    child_ids: (motion_category/parent_id)[]